resources/views/airlines.blade.php

```diff
@@ -0,0 +1,311 @@
+
+            const targetProperties = ["id", "name", "description"]
+
+            targetProperties.map((prop) => newRow.appendChild(createCell(airline[prop])))
+
+            newRow.appendChild(createCell(0))
+
+            newRow.appendChild(createEditDeleteBtns(airline.id))
+            createRow.after(newRow)
+        }
+
+        function createCell(content) {
+            const cellClass = document.querySelector("td").getAttribute("class")
+
+            let newCell = document.createElement("td")
+            newCell.setAttribute("class", cellClass)
+            newCell.textContent = content
+
+            return newCell
+        }
+
+        function createEditDeleteBtns(id) {
+            const cell = createCell("")
+            const container = document.createElement("div")
+            container.setAttribute("class", "flex flex-row")
+            container.setAttribute("id", "btn-container")
+
+            const editBtn = document.createElement("button")
+            editBtn.textContent = "Edit"
+            editBtn.setAttribute(
+                "class",
+                `edit-btn m-1 w-3/5 dark:bg-indigo-600 hover:bg-indigo-400
+                text-white font-semibold py-2 px-4 text-white rounded-xl`
+            )
+
+            const deleteBtn = document.createElement("button")
+            deleteBtn.textContent = "Delete"
+            deleteBtn.setAttribute(
+                "class",
+                `del-btn m-1 w-3/5 dark:bg-red-600 hover:bg-red-400
+                text-white font-semibold py-2 px-4 text-white rounded-xl`
+            )
+
+            container.append(editBtn)
+            container.append(deleteBtn)
+
+            cell.append(container)
+
+            return cell
+        }
+
+        async function createAirline() {
+            const name = document.querySelector('input[name="name"]')
+            const description = document.querySelector('input[name="description"]')
+
+            const newAirline = {
+                name: name.value.trim(),
+                description: description.value.trim()
+            }
+
+            const response = await fetch("/airlines", {
+                method: "POST",
+                headers: {
+                    "Accept": "application/json",
+                    "Content-Type": "application/json",
+                    "X-CSRF-TOKEN": document.querySelector('meta[name="csrf-token"]').content
+                },
+                body: JSON.stringify(newAirline)
+            })
+
+            if (! response.ok) {
+                const errors = await response.json()
+                const content = parseErrors(errors)
+                return displayModal("Creation Failed", content, "", "bg-red-500")
+            }
+
+            name.value = ""
+            description.value = ""
+
+            const storedAirline = await response.json()
+            return insertAirlineRow(storedAirline)
+        }
```

Comment:
would be nice to separate responsabilitites here. 
- get values from input
- make API call
- reset form